### DISCUSSION TITLE
"Bus" Object Type Not Detecting a School Bus?

### DISCUSSION THREAD

**Discussion by qJake** [blakeblackshear/frigate#4306] [Ask A Question]
I just finished setting up my Frigate environment with a Coral USB Accelerator and everything is working great, except one scenario. I want to be able to detect a school bus outside my house to know when my kids get home.

I can confirm that general object detection is working, and the Coral is processing fast (my inference speed is right around 8, and I have 6 1080p cameras connected).

So why isn't it detecting a school bus as a bus? I have the bus object type enabled and it's not behind any masks, zones, object filters, or anything else like that. It does cars on the same street just fine.

Does the model that ships with Frigate (in the official Docker container) just out of date? I was reading here:

https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet/edgetpu

Should I try one of these updated models? Has anyone else had any experience with this?

**Comment by NickM-27** [comment]
Bus has very few entries in the coco dataset so it's not surprising. https://user-images.githubusercontent.com/569905/152892743-f8b8648d-0df8-4307-b4a9-b27c22d9660c.png

In my experience buses like school buses actually just got detected as `car` so I'm surprised you haven't seen either. 

Might be worth lowering the `threshold` for bus and see if that is more effective.
